lib/galaxy/dependencies/pipfiles/default/Pipfile

```diff
@@ -50,6 +50,8 @@ pyparsing = "*"
 "Fabric3" = "*"
 paramiko = "*"
 python-genomespaceclient = "*"
+social_auth_core = "==1.5.0"
+pyjwkest = "==1.4.0"
```

Comment:
@VJalili Out of curiosity, why is `pyjwkest` also needed? I don't see it imported anywhere.

Reply:
It is required to _unpack_ some OIDC-based messages when handling callback from an OIDC IdP (e.g., authorization [`code`](https://developers.google.com/actions/identity/oauth2-code-flow#token_creation)). It is internally required by PSA ([see this requirements file](https://github.com/python-social-auth/social-core/blob/master/requirements-openidconnect.txt)); however, the `make update-dependencies` command did not capture it, so I had to add it manually.

Reply:
Ah, I see, I've fixed it in PR https://github.com/galaxyproject/galaxy/pull/5877 .

Reply:
Thanks.